### ISSUE TITLE
Error in Microsoft.Azure.Cosmos.csproj

### ISSUE THREAD

**Issue by fuocor** [Azure/azure-cosmos-dotnet-v3#984]
There is an error in the csproj file that prevents the propagation of the ```DefinedConstants```

```diff
  <PropertyGroup>
    <DefineConstants Condition=" '$(IsNightly)' == 'true' or '$(IsPreview)' == 'true' ">$(DefineConstants);PREVIEW</DefineConstants>
    <DefineConstants Condition=" '$(SignAssembly)' == 'true' ">$(DefineConstants);SignAssembly</DefineConstants>
-    <DefineConstants Condition=" '$(DelaySign)' == 'true' ">DelaySignKeys</DefineConstants>
+    <DefineConstants Condition=" '$(DelaySign)' == 'true' ">$(DefineConstants);DelaySignKeys</DefineConstants>
    <DefineConstants>$(DefineConstants);DOCDBCLIENT;COSMOSCLIENT;NETSTANDARD20</DefineConstants>
  </PropertyGroup>
```

**Comment by ealsur** [comment]
@ausfeldt What do you think about this one?

**Comment by ausfeldt** [comment]
this was already merged in PR #1011

**Comment by ealsur** [comment]
Closing as PR with fix is already merged
